src/components/backend-ai-session-list.ts

```diff
@@ -1120,8 +1120,8 @@ export default class BackendAiSessionList extends BackendAIPage {
           menu.appendChild(mountedFolderItem);
         }
       });
+      document.body.appendChild(menu);
     }
```

Comment:
How about add 'else' part and add some description about 'only one folder is attached' status?